apl/tabular-operators/extend-valid-operator.mdx

```diff
@@ -1,60 +1,155 @@
 ---
-title: 'extend-valid'
-description: 'Creates calculated columns that are used to filter out rows. If valid it then appends them to the result set.'
-tags:
-    ['axiom documentation', 'documentation', 'axiom', 'tabular operators', 'extend-valid']
+title: extend-valid
+description: 'This page explains how to use the extend-valid operator in APL.'
 ---
 
-Creates calculated columns that are used to filter out rows. If valid it then appends them to the
-result set.
+The `extend-valid` operator in Axiom Processing Language (APL) allows you to extend a set of columns with new calculated values, where these calculations are based on conditions of validity for each row. It’s particularly useful when working with datasets that contain missing or invalid data, as it enables you to calculate and assign values only when certain conditions are met. This operator helps you keep your data clean by applying calculations to valid data points, and leaving invalid or missing values untouched.
```

Comment:
We have already used the abbreviation APL in the description text so feels unnecessary to use the full Axiom Processing Language name in these opening blurbs